R/theme.R

```diff
@@ -724,6 +724,19 @@ combine_elements <- function(e1, e2) {
     return(e2)
   }
 
+  # Inheritance of rel objects
+  if (is.rel(e1)) {
+    # Both e1 and e2 are rel, give product as another rel
+    if (is.rel(e2)) {
+      return(rel(unclass(e1) * unclass(e2)))
+    }
+    # If e2 is a unit, return modified unit
+    if (is.unit(e2)) {
+      return(unclass(e1) * e2)
+    }
+    return(e1)
+  }
+
```

Comment:
We might want to build in a check for whether `e2` is numeric and apply rel to that, just for completeness

Reply:
I was today years old when I learned:

``` r
library(grid)
is.numeric(unit(1, "npc"))
#> [1] TRUE
is.numeric(unit(1, "npc") + unit(1, "cm"))
#> [1] FALSE
```

<sup>Created on 2023-09-12 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>